nanoprintf.h

```diff
@@ -62,8 +62,17 @@ NPF_VISIBILITY int npf_vpprintf(
 #ifndef NANOPRINTF_IMPLEMENTATION_INCLUDED
 #define NANOPRINTF_IMPLEMENTATION_INCLUDED
 
-#include <inttypes.h>
+#include <limits.h>
 #include <stdint.h>
+#include <inttypes.h>
+
+// The conversion buffer must fit at least UINT64_MAX in octal format with the leading '0'.
+#ifndef NANOPRINTF_CONVERSION_BUFFER_SIZE
+  #define NANOPRINTF_CONVERSION_BUFFER_SIZE    23
+#endif
+#if NANOPRINTF_CONVERSION_BUFFER_SIZE < 23
+  #error The size of the conversion buffer must be at least 23 bytes.
+#endif
```

Comment:
It will be nice to take nanoprintf up to C11 someday and start using `_Static_assert` :)